# Generated for NaoQi method ALFrameManager.playBehavior
# Starts a behavior

# The id of the box.
string id
---


